### PR TITLE
avoid re-install of ansible, upgrade ucp version to 1.0.0

### PR DESCRIPTION
**Pull request by vvb**
we try to re-install ansbile when the task runs as a part of another host-group. This commits will prevent it.
older versions of UCP are not available anymore - hence moving to 1.0.0

### CODE REVIEW

**Comment by vvb** [comment]
ping @mapuri this is required for the UCP, because of the 1.0.0 changes. PTAL.

File: roles/base/tasks/redhat_tasks.yml [hunk 1]
```diff
@@ -39,7 +39,15 @@
   shell: rm -rf /usr/lib64/python2.7/site-packages/pycrypto-*.egg-info
   when: '"Error unpacking rpm package python2-crypto-" in python_crypto_result.msg'
 
+- name: download ansible binary (redhat)
```
Comment (mapuri) [inline roles/base/tasks/redhat_tasks.yml]: just a minor nit, I think task name should be `download ansible rpm (redhat)`

**Comment by mapuri** [comment]
@vvb LGTM, just one minor comment.

**Comment by vvb** [comment]
@mapuri took care of the comment.. Thanks.

**Pull request merged**